pkg/kubelet/prober/worker.go

```diff
@@ -130,7 +130,10 @@ func (w *worker) run() {
 
 	// If kubelet restarted the probes could be started in rapid succession.
 	// Let the worker wait for a random portion of tickerPeriod before probing.
-	time.Sleep(time.Duration(rand.Float64() * float64(probeTickerPeriod)))
+	jitter := time.Duration(rand.Float64() * float64(probeTickerPeriod))
+	if jitter > time.Since(w.probeManager.start) {
```

Comment:
this is a strange check. When dice gives a small number - we do not wait even a very small timeout. If dice is giving a big timeout, we delay execution for that longer timeout. Would the check give better results if it will compare:

```suggestion
	if probeTickerPeriod > time.Since(w.probeManager.start) {
```

Reply:
Yeah, agreed, at the end of the day, the check only means don't sleep if the kubelet as started for long enough to disable the jitter. I will make that change and improve comments.